po/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+find_package(KF5I18n)
+set(LOCALE_INSTALL_DIR /usr/share/locale/)
```

Comment:
It is by default SET to /share/locale
So, if that line is removed, mo files will be installed at share/locale/xx/LC_MESSAGES/ which is not the correct path to connect catalogs.